projects/core/src/user/store/reducers/delivery-countries.reducer.ts

```diff
@@ -8,7 +8,7 @@ export const initialState: DeliveryCountriesState = {
 
 export function reducer(
   state = initialState,
-  action: fromAction.DeliveryCountriesAction | fromAction.MiscsDataAction
+  action: fromAction.DeliveryCountriesAction
```

Comment:
What about `MiscDataAction`? Wasn't it used before to clear countries state?

Reply:
Good catch